tools/rna_tools/targetfinder/tool_dependencies.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="targetfinder" version="1.07">
+        <install version="1.0">
+            <actions>
+                <action type="shell_command">git clone https://github.com/carringtonlab/targetfinder.git</action>
```

Comment:
Download the github release if you can:
https://github.com/carringtonlab/TargetFinder/archive/v1.7.tar.gz

With this we don't have a dependency on git and do not need to clone the entire repository. (This repo is small but it can matter for large ones.)